_contentTemplates/dropdowns/templates.md

```diff
@@ -0,0 +1,15 @@
+#item-template
+The `ItemTemplate` determines how the individual items are rendered in the dropdown element of the component. By default, the text from the particular suggestions is rendered.
```

Comment:
"dropdown element" is not very technically precise, so why not just use "dropdown" ?

Reply:
Fixed.